### ISSUE TITLE
documentation/example needed 

### ISSUE THREAD

**Issue by mnhan3** [BattletechModders/ModTek#68]
Some documentation is need on how to modify/add/delete things from json create by another mod without directly editing the json of that mod.   For example:   Mod-A create a number of new chasis.   It is loaded by modtek successfully.    I want to create a mod to modify things in the created chasis of Mod-A, how do i do that ?  is that even possible?   currently modtek tells me the json i'm modifying in my mod can't be found in the existing VersionManifest, even if I put my mod to load after Mod-A.  I've even tried setting Mod-A as a dependency.  Is there a way to add/modify/delete things within chasis created by mod-A in my mod that loads after Mod-A?  If so can an example be documented on how to do this?  I can easily modify vanilla defined chasis.

**Comment by Mpstark** [comment]
You'll need to provide an example of what you're trying to do. It's definitely possible to both replace an existing mods json files with your own file (just specify the file or a directory with file contained in manifest) or merge on top of another mod's JSON (add `"ShouldMergeJson": true` to the manifest entry).

**Comment by mnhan3** [comment]
I am trying to modify the hardpoints of some variants from another mod.   I do not want to directly modify the json of the mod if I can help it.   So lets use a real example,  I have downloaded the JK Variants.   I can install that mod and get it working with my campaign.    Now if I wanted to modify the hardpoints of one of the variants (ie. chassisdef_shadowhawk_SHD-2K).   I want to change the hardpoint for the head.    In vanilla if I wanted to modify a mech's hardpoint I would create a  json file with the same name in my mod under streamingassets/data/chassis containing just the array section I want to modify and it will be merged.    If I tried the same thing with the shadowhawk_SHD-2K which isn't part of vanilla, i get a error in the log "Could not find an existing VersionManifest entry for shadowhawk_SHD-2K".   To modify the hardpoint for another mod's chasis, what is required?  If I add my partial json to be loaded via the manifest section with "ShouldMergeJson": true, i get an endless startup screen with a counting number.    I understand I have to load my mod after jk-variant or have it as a dependency, but I can't seem to find the right combination to get the hardpoint changes to work.    I think I need a push in the right direction.    Hardpoint changes are one of a few modification I would like to make to some mods.  So figuring this out should go a long way to modifying some of the mods for what I want without directly changing their json files.
